packages/teleterm/src/mainProcess/mainProcess.ts

```diff
@@ -104,11 +116,52 @@ export default class MainProcess {
     });
   }
 
+  private resolveNetworkAddress(
```

Comment:
Looks good, though the only missing thing is some kind of a timeout mechanism. Otherwise the initialization process can hang forever waiting for the process to output the matching string.

Reply:
Done, I set 15s timeout, should be fine?

Reply:
> Done, I set 15s timeout, should be fine?

Yeah, I think even 10s would be enough. The server should get spun up pretty quickly, like within the first second or two so at 15 I feel like we're just making the user wait unnecessarily. ;)

Reply:
Done